apps/settings/elements/languages.html

```diff
@@ -29,6 +29,11 @@ <h1 data-l10n-id="language-header"> Language </h1>
             </select>
           </span>
         </li>
+        <li>
+          <a class="menu-item" id="more-languages" href="#">
+            <span data-l10n-id="more-languages"></span>
```

Comment:
Can you help to take off content from elements with `data-l10n-id` ? (for example : `sample-format`, `region` ... )